index.html

```diff
@@ -12,13 +12,24 @@
 </head>
 
 <body>
-    <div id="app">
+<div id="app">
         <about></about>
-        <navbar v-bind:suites="suites" v-if="loaded"></navbar>
-        <template v-for="(suite,index) in suites">
-            <task-suite v-bind:suite="suite" v-bind:index="index"></task-suite>
-        </template>
-    </div>
+        <div class="r_flex_container">
+            <div class="r_flex_fixed_child_top">
```

Comment:
By convention, all css classes and ids should be separated using `-`

For example: `r-flex-fixed-child-top`